cmd/mdatagen/loader.go

```diff
@@ -65,6 +65,10 @@ func (mvt *ValueType) UnmarshalText(text []byte) error {
 		mvt.ValueType = pcommon.ValueTypeBool
 	case "bytes":
 		mvt.ValueType = pcommon.ValueTypeBytes
+	case "[]string":
+		mvt.ValueType = pcommon.ValueTypeSlice
+	case "map[string]string":
+		mvt.ValueType = pcommon.ValueTypeMap
```

Comment:
```suggestion
	case "slice":
		mvt.ValueType = pcommon.ValueTypeSlice
	case "map":
		mvt.ValueType = pcommon.ValueTypeMap
```

Reply:
Yup.  I'll update this as well as the associated data type names in the corresponding example metric yaml file.